### ISSUE TITLE
Create mock weight sensor

### ISSUE THREAD

**Issue by patrickanguyen** [zotbins/ZotbinsCore#12]
## Description
Create class `MockWeight` which is a mock distance sensor that inherits from `IWeight`. We will use our mock sensor to create unit tests without depending on the actual hardware.

Depends on #11 

```mermaid
classDiagram
    IWeight <-- MockWeight
    class IWeight {
        +int32_t getWeight()
    }
    class MockWeight {
        -int32_t mWeight
    }
            
```

## Requirements
- Create a files in `lib/Weight called `MockWeight.hpp` and MockWeight.cpp
- Create the `MockWeight` class that inherits from IWeight and use `final` specifier so no class can inherit from it
- Create the constructor that takes in int32_t and set mWeight to it
- Override getWeight() and return mWeight

**Comment by patrickanguyen** [comment]
Resolved in #31

**Comment by patrickanguyen** [comment]
Resolved in #31
